src/main/webapp/onelive/index.html

```diff
@@ -277,13 +285,75 @@ <h5 class="title">
     </div>
 </script>
 
+<script id="upcoming-events-template" type="text/html">
+    <div class="col-md-10 offset-1">
+        <h3>Upcoming</h3>
+    </div>
+
+    {{#data}}
+    <div class="col-md-10 offset-1">
+        <a href="{{facebookLink}}" target="_blank">
+            <div class="card card-event shadow shadow-lg--hover mb-1 text-body">
+                <div class="card-body">
+                    <span class="h5 mr-1 card-event-date">{{date}}</span>
+                    <span class="small text-muted mr-4 text-uppercase">{{month}}, {{day}}</span>
+                    <span class="mr-5"> {{startTime}} - {{endTime}}</span>
+                    <span class="font-weight-bold ml-1">{{summary}}</span>
+                </div>
+            </div>
+        </a>
+    </div>
+    {{/data}}
+</script>
+
 <script>
     $(function () {
         loadNavAndFooter('/assets/content/static');  //relative path to content directory
+        loadUpcomingEvents(); //load upcoming events
         loadProfiles(); // load profiles
         loadFaqComponent();
     });
 
+    //function to load upcoming events
+    function loadUpcomingEvents() {
+        $.ajax({
+            type: 'GET',
+            url: 'http://localhost:8080/calendar/onelive',
+            dataType: 'json',
+            success: function (payload) {
+                if (payload.items == null) {
+                    $("#section-events").hide();
+                } else {
+                    for (let i = 0; i < payload.items.length; i++) {
+                        //to get img link (optional)
+                        /*let fileId = payload.items[i].attachments[0].fileId;
+                        payload.items[i].imgLink = "https://drive.google.com/uc?export=view&id=" + fileId;*/
+
+                        //get fb link and remove it from description (split the part of the description where the fb link is added)
+                        let description = payload.items[i].description;
+                        payload.items[i].facebookLink = description.substring(description.lastIndexOf('<a href') + 9, description.lastIndexOf('" id='));
+                        payload.items[i].description = description.substring(0, description.lastIndexOf('<a href'));
+                        //get date & time of the event
+                        let start = new Date(payload.items[i].start.dateTime);
+                        let end = new Date(payload.items[i].end.dateTime);
+                        payload.items[i].day = start.toString().substring(0, 3);
+                        payload.items[i].month = start.toString().substring(4, 7);
+                        payload.items[i].date = start.toString().substring(8, 10);
+                        payload.items[i].startTime = start.toString().substring(16, 21);
+                        payload.items[i].endTime = end.toString().substring(16, 21);
```

Comment:
Try .toLocaleDateString() and .toLocaleTimeString()

Reply:
Not necessary but check those methods out.

Reply:
Ok I'll check

Reply:
It works! Thanks :star_struck: